frontend/locales/fr.ts

```diff
@@ -1,7 +1,864 @@
+            info: 'Catégorie: {0}',
+            admin_tools: "Outils d'administration",
+            chat: 'Chat',
+            dev_tools: 'Outils de développeur',
+            economy: 'Economie',
+            gameplay: 'Jouabilité',
+            games: 'Jeux',
+            protection: 'Protection',
+            role_playing: 'Jeux de rôles',
+            world_management: 'Gestion du monde',
+            misc: 'Divers',
+        },
+        actions: {
+            unwatch: "Arrêter d'observer",
+            watch: 'Observer',
+            flag: 'Alerter',
+            star: 'Etoiler',
+            unstar: 'Désétoiler',
+            adminActions: "Actions de l'administrateur",
+            flagHistory: 'Historique des alertes ({0})',
+            staffNotes: 'Notes du personnel ({0})',
+            userActionLogs: "Journaux d'actions utilisateur",
+            forum: 'Forum',
+        },
+        flag: {
+            flagProject: 'Alerte {0} ?',
+            flagSend: "Signalé avec succès, merci d'avoir fait de cette communauté un endroit meilleur !",
+            flagSent: 'Alerte soumise pour examen',
+            flags: {
+                inappropriateContent: 'Contenu inapproprié',
+                impersonation: "Usurpation d'identité ou tromperie",
+                spam: 'Pourriel',
+                malIntent: 'Malveillance',
+                other: 'Autre',
+            },
+            error: {
+                alreadyOpen: "Vous n'avez seulement qu'une seule alerte non résolue sur un projet",
+                alreadyResolved: 'Cette alerte est déjà résolue',
+            },
+        },
+        tabs: {
+            docs: 'Documentations',
+            versions: 'Versions',
+            discuss: 'Discussion',
+            settings: 'Paramètres',
+            homepage: "Page d'acceuil",
+            issues: 'Problèmes',
+            source: 'Source',
+            support: 'Support',
+        },
+        new: {
+            step1: {
+                title: "Accord de l'utilisateur",
+                text: 'Un projet contient vos téléchargements et la documentation pour votre plugin.<br>Avant de continuer; veuillez consulter les <a href="#">directives de soumission à Hangar.</a>',
+                continue: 'Accepter',
+                back: 'Annuler',
+            },
+            step2: {
+                title: 'Paramètres basics',
+                continue: 'Continuer',
+                back: 'Retour',
+                userSelect: 'Créer comme...',
+                projectName: 'Nom du projet',
+                projectSummary: 'Résumé du projet',
+                projectCategory: 'Catégorie du projet',
+            },
+            step3: {
+                title: 'Paramètres additionnels',
+                continue: 'Continuer',
+                back: 'Retour',
+                optional: 'Optionel',
+                links: 'Liens',
+                homepage: "Page d'acceuil",
+                issues: 'Traqueur de problèmes',
```

Comment:
optionnel: updated in my fork
accueil: same
issues: I have the feeling that mine is better but I'll leave it as unresolved so people will debate.

Reply:
[Gitea](https://github.com/go-gitea/gitea) and other projects translate *issue* with *ticket*, since *problème* hardly describes a question or a suggestion. There is no exact and exhaustive translation. :confused: